.github/workflows/test.yml

```diff
@@ -54,9 +57,7 @@ jobs:
         echo "SELECT version()" | psql -h database -U username testdb
         echo "SELECT CURRENT_TIME" | psql -h database -U username testdb
 
-    - name: Setup
-      run: |
-        ./bin/setup
+    - run: bundle install
```

Comment:
I removed the usage of `./bin/setup` as it was installing both gemfiles, which we don't want when not using gemika imo.

Reply:
That's a good point! I like to keep `bin/setup` as the no-brainer solution for getting it up and running locally.